app/frontend/app/templates/anime/index.hbs

```diff
@@ -233,6 +233,7 @@
   <!-- Left side End -->
 
   <!-- Right Side Start -->
+  {{#unless notYetAired}}
   <div class="community-column">
```

Comment:
`div` and the following elements should probably be indented too to follow the layout already used:

```hbs
{{#if roundedBayesianRating}}
       <div class="hb-score">
              ...
       </div>
{{/if}}
```